main.html

```diff
@@ -284,6 +284,23 @@ <h2 class="about-header">About Myself</h2>
             </ul>
           
         </section>
+
+        <section class="contact">
+            <h2 class="contact-header">Contact me</h2>
+            <p class="contact-par">If you have an application you are interested in developing, a feature that you need built or
+                a project that needs
+                coding. I’d love to help with it</p>
+            <form class="contact-form" action="https://formspree.io/f/myyazwzw" method="POST">
+                <ul class="con-f-list">
+                    <li><input class="name" type="text" name="username"  placeholder="Yeremias" maxlength="30" required></li>
+                    <li><input class="email"  type="email" name="useremail"  placeholder="natajayanj14@gmail.com" required></li>
+                    <li><textarea class="feedback" name="userfeedback" placeholder="Write your message here" cols="30" rows="10" maxlength="500"
+                            required></textarea></li>
+                    <li><button class="form-button">Get It Touch</button></li>
+                </ul>
+            </form>
+        
+        </section>
```

Comment:
- [ ] The previous reviewer rightly pointed out the need to make it responsive on the mobile view. you did a good job with the design, but it's fixed I've attached a screenshot to show you how it looks, and the equivalent on figma. kindly adjust your CSS.

# Your design :
![image](https://user-images.githubusercontent.com/50990925/213123929-dba953cf-cbbc-4ba9-9842-02cfb21c7bca.png)

# Figma :
![image](https://user-images.githubusercontent.com/50990925/213124375-b369ed63-0730-4dfa-b416-a3c4bf5e08f4.png)

Reply:
@Shedrack-Sunday  So the width should still be 375px.

Reply:
Yes, please. Sorry for the late reply